src/itemlist_formaction.cpp

```diff
@@ -84,9 +84,14 @@ void itemlist_formaction::process_operation(operation op, bool automatic, std::v
 		LOG(level::INFO, "itemlist_formaction: opening item at pos `%s'", itemposname);
 		if (itemposname.length() > 0 && visible_items.size() != 0) {
 			if (itempos < visible_items.size()) {
+
+				bool browser_success = v->open_in_browser(visible_items[itempos].first->link());
+				if (!browser_success)
```

Comment:
This still doesn't display the error to the user. Do you need any help with this?